torch/csrc/jit/codegen/cuda/transform_replay.cpp

```diff
@@ -279,14 +320,8 @@ std::pair<TensorDomain*, unsigned int> TransformReplay::replayPasC(
   for (auto c_id : target_consumer_ids) {
     auto it = replay_PasC.getReplay().find(c_id);
     if (it == replay_PasC.getReplay().end()) {
-      // check if c_id depends on selected domain
-      auto selected_domain =
-          ir_utils::getSelectedDomainIfTvIsIndexSelectOutput(consumer);
```

Comment:
Is `getSelectedDomainIfTvIsIndexSelectOutput` still used anywhere else? If not, we need to clean it up.

Reply:
No, I don't think so. Will remove.